translations/tugas_kelompok.html

```diff
@@ -0,0 +1,63 @@
+<!-- Title: Group Assignment -->
+
+<h3>Description</h3>
+<p style="text-align: justify;">
+  Feli is a student at a popular university. One day, Feli is given a group assignment consisting
+  of $$$N$$$ questions. The $$$i$$$-th question can be solved in $$$X_i$$$ amount of time. Because
+  the group assignment has many questions, Feli formed a group of $$$M$$$ people
+</p>
+<p style="text-align: justify;">
+  To complete the group assignment, Feli's group agrees to split work among them. Each member
+  chooses two integers $$$L$$$ and $$$R$$$, and the member has to solve questions from $$$L$$$ to
+  $$$R$$$ inclusive.
+</p>
+<p style="text-align: justify;">
+  With this method, each question can be solved by more than one member. The questions, however,
+  remain at the same difficulty so that the time needed to solve the $$$i$$$-th question is still
+  $$$X_i$$$. The time it takes for a member to solve the questions is equal to the sum of the time
+  it takes to solve each question.
+</p>
+<p style="text-aligh: justify;">
+  Feli's group wants to minimize the amount of the time they spend on solving the questions. Help
+  Feli's group determine the minimum time they can spend!
+</p>
+
+<h3>Input Format</h3>
+<p style="text-align: justify;">
+  Input starts with a positive integer $$$T$$$, the number of test cases, and the next lines will
+  be explained below.
+</p>
+<p style="text-align: justify;">
+  The first line consists of two positive integer $$$N$$$ and $$$M$$$ - the number of questions
+  and Feli's friends. The next line consists of $$$N$$$ integer $$$X_i$$$, the time it takes to
+  solve the $$$i$$$-th question.
+</p>
+
+<h3>Output Format</h3>
+<p style="text-align: justify;">
+  For each test case, output "Case #$$$X$$$: $$$Y$$$", where $$$X$$$ is the case number (starting
+  from 1), and $$$Y$$$ is the time it takes to complete the assignment.
+</p>
+
+<h3>Constraint</h3>
+<ul>
+  <li style="text-align: justify;">$$$1 &le; T &le; 100$$$</li>
+  <li style="text-align: justify;">$$$1 &le; N,M &le; 10000$$$</li>
+  <li style="text-align: justify;">$$$1 &le; L &le; R &le; N$$$</li>
+  <li style="text-align: justify;">$$$1 &le; X_i &le; 10^7$$$</li>
+</ul>
+
+<h3>Sample Input</h3>
+<pre>
+  2
+  3 2
+  3 1 1
+  3 2
+  3 2 1
+</pre>
+
+<h3>Sample Output</h3>
+<pre>
+  Case #1: 3
+  Case #2: 3
```

Comment:
No need indent. Spaces are not ignored inside the `<pre>` tag
```suggestion
Case #1: 3
Case #2: 3
```